spec/controllers/videos_controller_spec.rb

```diff
@@ -27,8 +27,8 @@
         get :show, id: video.id
       end
 
-      it 'sets the @video variable' do
-        expect(assigns(:video)).to eq(video)
+      it 'sets the @video to the VideoDecorator' do
+        expect(assigns(:video)).to be_decorated_with VideoDecorator
```

Comment:
Yes, that's the draper gem in action. 
I'm not sure yet if I like or dislike the gem for hiding the complexity of decorators.